src/tools/cephfs/cephfs-shell

```diff
@@ -21,11 +21,14 @@ try:
 except ImportError:
     def with_argparser(argparser):
         import functools
+
         def argparser_decorator(func):
             @functools.wraps(func)
             def wrapper(thiz, cmdline):
-                # do not split if it's already a list
-                if not isinstance(cmdline, list):
+                if isinstance(cmdline, list):
+                    arglist = cmdline
+                else:
+                    # do not split if it's already a list
                     arglist = shlex.split(cmdline, posix=False)
```

Comment:
@kefu Just a small doubt. This should only remove the outer quotes of string right? Does this hold good if there are quotes inside the string?

Reply:
per the doc string:
```
In [1]: ?str.strip
Signature: str.strip(self, chars=None, /)
Docstring:
Return a copy of the string with leading and trailing whitespace remove.
```

so, i think it only removes the leading and trailing quotes. and the following comment put. 

> Does this hold good if there are quotes inside the string?

i think so. but why don't you give it a try?

Reply:
```
>>> str = 'te"st"'
>>> str.strip('\'""') 
'te"st' 
```
I was just looking at a scenario like this.
it should acutally return `te"st"` right?

Reply:
````
def dequote(string):
    if len(string) > 1 and string[0] == string[-1] and string[0] in ['"', "'"]:
        string = string[1:-1]
    return string
````
Something like this works.

Reply:
yeah, strictly speaking, we should not accept unbalanced quotes in command line arguments.

Reply:
thanks for sharing your thoughts. but i don't think it's in the scope of my PR.

Reply:
Ya. Thanks.